lib/dateTime.js

```diff
@@ -0,0 +1,578 @@
+			days[1] = days[2] = ['일', '월', '화', '수', '목', '금', '토'];
+			break;
+		case 'nl':
+			dateFormats = ['dddd d MMMM yyyy', 'd MMMM yyyy', 'dd-MM-yyyy', 'MMMM yyyy', 'd MMMM'];
+			dayPeriods = ['a.m.', 'p.m.'];
+			months = [numericMonths, numericMonths];
+			days = [
+				['zondag', 'maandag', 'dinsdag', 'woensdag', 'donderdag', 'vrijdag', 'zaterdag'],
+				['zo', 'ma', 'di', 'wo', 'do', 'vr', 'za'],
+				['Z', 'M', 'D', 'W', 'D', 'V', 'Z']
+			];
+			break;
+		case 'pt':
+			dateFormats = ['dddd, d\' de \'MMMM\' de \'yyyy', 'd\' de  \'MMMM\' de \'yyyy', 'd/M/yyyy', 'MMMM\' de \'yyyy', 'dd\' de \'MMMM'];
+			months = [
+				['janeiro', 'fevereiro', 'março', 'abril', 'maio', 'junho', 'julho', 'agosto', 'setembro', 'outubro', 'novembro', 'dezembro'],
+				['jan', 'fev', 'mar', 'abr', 'mai', 'jun', 'jul', 'ago', 'set', 'out', 'nov', 'dez']
+			];
+			days = [
+				['domingo', 'segunda-feira', 'terça-feira', 'quarta-feira', 'quinta-feira', 'sexta-feira', 'sábado'],
+				['dom', 'seg', 'ter', 'qua', 'qui', 'sex', 'sáb'],
+				['D', 'S', 'T', 'Q', 'Q', 'S', 'S']
+			];
+			break;
+		case 'sv':
+			dateFormats = ['dddd \'den\' d MMMM yyyy', 'd MMMM yyyy', 'yyyy-MM-dd', 'MMMM yyyy', 'dd MMMM'];
+			dayPeriods = ['fm', 'em'];
+			months = [
+				['Januari', 'Februari', 'Mars', 'April', 'Maj', 'Juni', 'Juli', 'Augusti', 'September', 'Oktober', 'November', 'December'],
+				['Jan', 'Feb', 'Mar', 'Apr', 'Maj', 'Jun', 'Jul', 'Aug', 'Sep', 'Okt', 'Nov', 'Dec']
+			];
+			days = [
+				['Söndag', 'Måndag', 'Tisdag', 'Onsdag', 'Torsdag', 'Fredag', 'Lördag'],
+				['Sön', 'Mån', 'Tis', 'Ons', 'Tor', 'Fre', 'Lör'],
+				['S', 'M', 'T', 'O', 'T', 'F', 'L']
+			];
+			break;
+		case 'tr':
+			dateFormats = ['dd MMMM yyyy dddd', 'dd MMMM yyyy', 'dd.MM.yyyy', 'MMMM yyyy', 'dd MMMM'];
+			dayPeriods = ['ÖÖ', 'ÖS'];
+			months = [
+				['Ocak', 'Şubat', 'Mart', 'Nisan', 'Mayıs', 'Haziran', 'Temmuz', 'Ağustos', 'Eylül', 'Ekim', 'Kasım', 'Aralık'],
+				['Oca', 'Şub', 'Mar', 'Nis', 'May', 'Haz', 'Tem', 'Ağu', 'Eyl', 'Ek', 'Kas', 'Ara']
+			];
+			days = [
+				['Pazar', 'Pazartesi', 'Salı', 'Çarşamba', 'Perşembe', 'Cuma', 'Cumartesi'],
+				['Paz', 'Pzt', 'Sal', 'Çar', 'Per', 'Cum', 'Cmt'],
+				['P', 'P', 'S', 'Ç', 'P', 'C', 'C']
+			];
+			break;
+		case 'zh':
+			dateFormats = ['yyyy年M月d日', 'yyyy年M月d日', 'yyyy/M/d', 'yyyy年M月', 'M月d日'];
+			dayPeriods = ['上午', '下午'];
+			months[0] = months[1] = ['一月', '二月', '三月', '四月', '五月', '六月', '七月', '八月', '九月', '十月', '十一月', '十二月'];
+			days[0] = days[1] = ['週日', '週一', '週二', '週三', '週四', '週五', '週六'];
+			days[2] = ['日', '一', '二', '三', '四', '五', '六'];
+			fullTimeFormat = `ZZZ ${timeFormat}`;
+			break;
+	}
+
+	switch (language) {
+		case 'en-gb':
+			dateFormats = ['dddd, d MMMM yyyy', 'dd MMMM yyyy', 'dd/MM/yyyy', 'MMMM yyyy', 'd MMMM'];
+			break;
+		case 'fr-ca':
+			dateFormats[1] = 'MMM d yyyy';
+			dateFormats[2] = 'yyyy-MM-dd';
+			dateFormats[4] = 'MMMM d';
+			firstDayOfWeek = 0;
+			break;
+		case 'zh-tw':
```

Comment:
Looks like this might also need a custom `days` array?

Reply:
Correct -- added that in, as well as tests for the missing formats. The reason this didn't get found is that none of the standard date formats use the long day for the ZH locale.